src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -153,15 +153,15 @@ class PrisonerIepLevelReviewService(
       )
     )
 
-    nextReviewDateUpdaterService.update(review.bookingId)
```

Comment:
This was done inside the `persistSyncPostRequest()` method. This method is used during the handling of both sync POST requests and also migration requests.

I've moved this call out so that it now only happen during the handling of sync POST requests (`handleSyncPostIepReviewRequest()` method).